lib/collection/src/shards/telemetry.rs

```diff
@@ -55,6 +56,7 @@ impl Anonymize for LocalShardTelemetry {
     fn anonymize(&self) -> Self {
         LocalShardTelemetry {
             variant_name: self.variant_name.clone(),
+            points_indexed_once: 0,
```

Comment:
Can we use `anonymize()` here?

```suggestion
            points_indexed_once: self.points_indexed_once.anonymize(),
```

Reply:
Done.